GDJS/Runtime/runtimescene.ts

```diff
@@ -830,27 +846,57 @@ namespace gdjs {
       if (
         syncedPlayerNumber !== undefined &&
         syncedPlayerNumber !== 1 &&
-        (!this.networkId ||
-          (variablesNetworkSyncData.length === 0 &&
-            !Object.keys(extensionsVariablesSyncData).length))
+        !this.networkId
       ) {
         // If we are getting sync data for a specific player,
-        // and they are not the host, there is no sync data to send if:
-        // - The scene has no networkId (it's either not a multiplayer scene or the scene is not yet networked).
-        // - There are no variables to sync in the scene or extensions.
+        // and they are not the host, there is no sync data to send if
+        // the scene has no networkId (it's either not a multiplayer scene or the scene is not yet networked).
         return null;
       }
 
-      return {
+      const networkSyncData: LayoutNetworkSyncData = {
         var: variablesNetworkSyncData,
         extVar: extensionsVariablesSyncData,
         id: this.getOrCreateNetworkId(),
       };
+      if (syncOptions.syncSceneTimers) {
+        networkSyncData.time = this._timeManager.getNetworkSyncData();
+      }
+      if (syncOptions.syncOnceTriggers) {
+        networkSyncData.once = this._onceTriggers.getNetworkSyncData();
+      }
+      // The function gdjs.evtTools.tween.getTweensMap may not exist if the project is not using any tweens,
+      // as it's an extension.
+      if (syncOptions.syncTweens && gdjs.evtTools.tween.getTweensMap) {
+        networkSyncData.tween = gdjs.evtTools.tween
+          .getTweensMap(this)
+          .getNetworkSyncData();
+      }
```

Comment:
Should that be a callback registered by the extension tween, something like `gdjs.registerRuntimeSceneGetSyncDataCallback`?

```suggestion
      gdjs.callbacksRuntimeSceneGetSyncData.forEach(callback => callback(this));
```